raiden_contracts/tests/test_channel_cooperative_settle.py

```diff
@@ -10,7 +11,7 @@
 from raiden_contracts.utils.events import check_channel_settled, check_withdraw_2
 from raiden_contracts.utils.type_aliases import BlockExpiration
 
-EXPIRATION = BlockExpiration(100)
+EXPIRATION = int(time.time()) + BlockExpiration(100) * 15
```

Comment:
Sorry for putting this comment at such a bad location, but does the type `BlockExpiration` still make sense this way? 🤔 Searching through its code reference doesn't seem like this is properly used anymore. Do you agree?